llpc/test/shaderdb/relocatable_shaders/TriangleVs_CheckNoteSectionForCacheHash.spvasm

```diff
@@ -1,3 +1,17 @@
+; This test case checks that the 128-bit cache hash is correctly added to the PAL metadata.
+; BEGIN_SHADERTEST
+; RUN: amdllpc -spvgen-dir=%spvgendir% -gfxip=9 -unlinked -enable-relocatable-shader-elf -v -o %t.elf %s | FileCheck --check-prefix=CHECK1 %s
+
+; CHECK1-LABEL: {{^// LLPC}} calculated hash results (graphics pipeline)
+; CHECK1-LABEL: Building pipeline with relocatable shader elf.
+; CHECK1-NEXT:  LLPC version: [[llpc_version:([0-9a-f]{2} ?){8}]]{{$}}
+; CHECK1:       Hash for vertex stage cache lookup: [[#%.2x,llpc_vert_hash_byte1:]] [[#%.2x,llpc_vert_hash_byte2:]] [[#%.2x,llpc_vert_hash_byte3:]] [[#%.2x,llpc_vert_hash_byte4:]] [[#%.2x,llpc_vert_hash_byte5:]] [[#%.2x,llpc_vert_hash_byte6:]] [[#%.2x,llpc_vert_hash_byte7:]] [[#%.2x,llpc_vert_hash_byte8:]] [[#%.2x,llpc_vert_hash_byte9:]] [[#%.2x,llpc_vert_hash_byte10:]] [[#%.2x,llpc_vert_hash_byte11:]] [[#%.2x,llpc_vert_hash_byte12:]] [[#%.2x,llpc_vert_hash_byte13:]] [[#%.2x,llpc_vert_hash_byte14:]] [[#%.2x,llpc_vert_hash_byte15:]] [[#%.2x,llpc_vert_hash_byte16:]]
+; CHECK1-LABEL: {{^// LLPC}} SPIRV-to-LLVM translation results
+
+; CHECK1:       128_bit_cache_hash: [ 0x[[#%.16X,llpc_vert_hash_byte1]] 0x[[#%.16X,llpc_vert_hash_byte2]] 0x[[#%.16X,llpc_vert_hash_byte3]] 0x[[#%.16X,llpc_vert_hash_byte4]] 0x[[#%.16X,llpc_vert_hash_byte5]] 0x[[#%.16X,llpc_vert_hash_byte6]] 0x[[#%.16X,llpc_vert_hash_byte7]] 0x[[#%.16X,llpc_vert_hash_byte8]] 0x[[#%.16X,llpc_vert_hash_byte9]] 0x[[#%.16X,llpc_vert_hash_byte10]] 0x[[#%.16X,llpc_vert_hash_byte11]] 0x[[#%.16X,llpc_vert_hash_byte12]] 0x[[#%.16X,llpc_vert_hash_byte13]] 0x[[#%.16X,llpc_vert_hash_byte14]] 0x[[#%.16X,llpc_vert_hash_byte15]] 0x[[#%.16X,llpc_vert_hash_byte16]] ]
```

Comment:
I haven't checked, but think you can drop the exact format specifier, e.g.,:
`0x[[#%.16X,llpc_vert_hash_byte1]]` -> `0x[[#%X,llpc_vert_hash_byte1]]`. I think in this test we'd want to check that the data is as expected, and we don't care about the exact output format.

Reply:
I tried, and it needs the specifier.